action.yml

```diff
@@ -38,6 +38,9 @@ inputs:
   service-now-password:
     description: the service account password
     required: false
+  environment:
+    description: the firebase environment to use (https://firebase.google.com/docs/functions/config-env)
+    required: false
```

Comment:
**thought**: this could be merged with project-id's description

Reply:
Oops. I forgot to remove this.